js/source/raster_tile_source.js

```diff
@@ -51,10 +51,13 @@ RasterTileSource.prototype = util.inherit(Evented, {
         function done(err, img) {
             delete tile.request;
 
-            if (tile.aborted)
-                return;
+            if (tile.aborted) {
+                this.state = 'unloaded';
+                return callback(null);
+            }
```

Comment:
Yes, technically, it's a seperate bug that manifested itself in the same way (preventing loaded() from being true.)

>  I'm guessing something along the lines of "map.loaded() doesn't return true if there are any tiles were aborted."

Yes, exactly.